src/jenkins_job_insight/main.py

```
@@ -2061,37 +2117,40 @@ async def create_github_issue_endpoint(
     )
-
-    if not tests_repo_url or not github_token:
+    if not github_token:
         raise HTTPException(
             status_code=400,
-            detail="TESTS_REPO_URL and GITHUB_TOKEN must be configured to create GitHub issues",
+            detail="GitHub token is required. Provide a token in your profile settings or configure GITHUB_TOKEN on the server.",
         )
 
-    # Verify classification matches tracker
     failure, _result_data = await _load_effective_failure(
```

Ruff (0.15.9)

[warning] Unpacked variable `failure` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)